src/test/java/rueppellii/backend2/tribes/gameUtility/timeService/TimeServiceTest.java

```diff
@@ -0,0 +1,111 @@
+package rueppellii.backend2.tribes.gameUtility.timeService;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import rueppellii.backend2.tribes.gameUtility.timeService.TimeService;
+import rueppellii.backend2.tribes.progression.persistence.ProgressionModel;
+
+import static rueppellii.backend2.tribes.gameUtility.timeService.TimeConstants.*;
+
+class TimeServiceTest {
+
+    @InjectMocks
+    private TimeService timeService;
+
+    private Integer buildingLevel;
+    private Integer townhallLevel;
+    private Integer troopLevel;
+    private Double troopUpgradeTimeMultiplier;
+    private Double troopCreationTimeMultiplier;
+    private ProgressionModel progressionModel;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
+        buildingLevel = 1;
+        townhallLevel = 1;
+        troopLevel = 1;
+        troopCreationTimeMultiplier = 1.0;
+        troopUpgradeTimeMultiplier = 1.0;
+    }
+
+    @Test
+    void calculateTimeOfBuildingCreationTest() {
```

Comment:
Try to follow this naming convention:
[the name of the tested feature][expected input / tested state][expected behavior]

like

registerNewUserAccount_ExistingEmailAddressGiven_ShouldThrowException()